src/client/App.scss

```diff
@@ -62,6 +62,16 @@ a {
   text-decoration: none;
 }
 
+input[type=radio]
+{
+  -ms-transform: scale(1.01); /* IE */
+  -moz-transform: scale(1.01); /* FF */
+  -webkit-transform: scale(1.01); /* Safari and Chrome */
+  -o-transform: scale(1.01); /* Opera */
```

Comment:
Can we remove all the vendor prefix versions? I don't think any functioning browser (mobile or otherwise) need it.

Reply:
Also, should we scope this fix to .disaster-question input[type=radio]?

Reply:
I'll update in a separate PR!